.travis.yml

```diff
@@ -50,7 +50,15 @@ matrix:
       env:
         - JOBCACHE=8
 
-    - name: publish documentation
+    - name: Lint Python code with flake8
+      language: python
+      python: 3.6
+      cache:
+        - pip
+      install: pip install flake8
+      script: flake8 . --count --exclude=./.*,./Lib,./vm/Lib  --select=E9,F63,F7,F82 --show-source --statistics
```

Comment:
What is the reason to lint the example snippets? Does this give us feedback? The snippets are not of high quality code. They are more intended to test certain python behaviors.

Reply:
I really like the snippets -- I nice test of Python syntax corner cases.  As I wrote above, I added a few __noqa__ directives where needed.  I think it is a good idea to lint the snippet so that contributors are warned when new test cases  contain syntax errors and undefined names.  It these are intentional then noqa can be added and if not the linter should improve the quality of submissions.

Awesome project!!